createtable.go

```diff
@@ -216,6 +217,18 @@ func (ct *CreateTable) Tag(key, value string) *CreateTable {
 	return ct
 }
 
+// SSEEncryption specifies the server side encryption for this table.
+// Encryption is disabled by default. The valid values for sseType are {AES256, KMS}.
+func (ct *CreateTable) SSEEncryption(enabled bool, keyID, sseType string) *CreateTable {
```

Comment:
Let's define a type for SSEType instead of taking any string.

Something like this:
https://github.com/guregu/dynamo/blob/a59ef6bb76bcd42fc2288000c844df9aea7d248e/keys.go#L3-L12

A lot of constants in this library are named ThingCategory instead of CategoryThing which is less than ideal, if that makes any sense. So maybe naming constants for the SSE stuff should have SSE as a prefix like `SSETypeAES256` and `SSETypeKMS`.